core-chart/templates/kubeflex-clusterrolebinding.yaml

```diff
@@ -0,0 +1,18 @@
+{{- range $cp := (concat (.Values.ITSes | default list) (.Values.WDSes | default list) ) }}
+{{- if eq $cp.type "host" }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: kubeflex-manager-cluster-admin-rolebinding
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin

```

Comment:
Is this much really necessary?
Cam be addressed in follow-on.

Reply:
it is necessary for host CPs. it will be installed only if host CP are being created